### PR TITLE
CNV-32607: remove TP from instance types

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.15+

Issue: [CNV-32607](https://issues.redhat.com//browse/CNV-32607)

Link to docs preview: https://68132--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by 0xFelix** [comment]
What is the correct way to spell instancetypes? AFAIK `InstanceTypes` was chosen to be the correct on for the UI? This is talking about `instance types again`.

At step 3 I would replace `the configuration appropriate for your workload` with `the resource sizing appropriate for your workload`. (an instancetype defines the amount of resources a VM will get while a preference configures runtime preferences the guest OS inside the VM expects, e.g. certain hardware aspects.)

**Comment by sabrinajess** [comment]
@0xFelix We only use `InstanceTypes `when we're referring specifically to the UI button/filter/section and not as a noun (in that case, we use` instance types`).

**Comment by 0xFelix** [comment]
@sabrinajess ACK, what about my second suggestion?

**Comment by sabrinajess** [comment]
@0xFelix Implementing the second one now. Meant to include that in my original comment!

**Comment by sabrinajess** [comment]
@0xFelix Please review and mark if it LGTY.

**Comment by 0xFelix** [comment]
gg

File: modules/virt-creating-vm-instancetype.adoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ The volume table only lists volumes in the `openshift-virtualization-os-images`
 ====
 
 ifdef::virt-create-vms[]
-. Click an instance type tile and select the configuration appropriate for your workload.
+. Click an instance type tile and select the resource sizing appropriate for your workload.
```
Comment (mburke5678) [inline modules/virt-creating-vm-instancetype.adoc:51]: This reads a bit better, in my mind; is it correct wording?
```suggestion
. Click an instance type tile and select the resource size appropriate for your workload.
```

**Comment by mburke5678** [comment]
@sabrinajess One nit. Otherwise LGTM

**Pull request merged**